src/main/java/de/qabel/desktop/ui/actionlog/ActionlogController.java

```diff
@@ -21,75 +34,175 @@
+
 	@Inject
 	ClientConfiguration clientConfiguration;
 
+	@Inject
+	private ContactRepository contactRepository;
+
+	Identity identity;
+	Contact c;
+	Date lastDate;
+	DropHTTP dHTTP = new DropHTTP();
 
 	public void initialize(URL location, ResourceBundle resources) {
 		try {
-			loadMessages();
-		} catch (EntityNotFoundExcepion entityNotFoundExcepion) {
+			identity = clientConfiguration.getSelectedIdentity();
+			c = new Contact(identity, identity.getAlias(), identity.getDropUrls(), identity.getEcPublicKey());
+			receiveDropMessages(null);
+
+			scroller.setVvalue(scroller.getVmax());
+			addChangeListener();
+
+		} catch (QblVersionMismatchException | QblDropInvalidMessageSizeException | QblSpoofedSenderException entityNotFoundExcepion) {
 			entityNotFoundExcepion.printStackTrace();
 		}
 	}
 
-	void loadMessages() throws EntityNotFoundExcepion {
-		messages.getChildren().clear();
-		createMessageItems();
-		if(scroller.getVmax() == scroller.getVvalue()) {
-			scroller.setVvalue(scroller.getVmax());
+	private void addChangeListener() {
+		((Region) scroller.getContent()).heightProperty().addListener((ov, old_val, new_val) -> {
+			if (scroller.getVvalue() != scroller.getVmax()) {
+				scroller.setVvalue(scroller.getVmax());
+			}
+		});
+	}
+
+	@FXML
+	protected void handleSubmitButtonAction(ActionEvent event) throws QblDropPayloadSizeException, EntityNotFoundExcepion {
+		DropMessage d = sendDropMessage(c, textarea.getText());
+		addOwnMessageToActionlog(d);
+	}
+
+	@FXML
+	protected void handleReloadMessagesButtonAction(ActionEvent event) {
+		try {
+			receiveDropMessages(lastDate);
+		} catch (QblDropInvalidMessageSizeException | QblVersionMismatchException | QblSpoofedSenderException e) {
+			e.printStackTrace();
 		}
 	}
 
-	private void createMessageItems() {
-		String text;
-		Map<String, Object> injectionContext = new HashMap<>();
-		text = "Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet.</html>";
-		injectionContext.put("text", text);
-		MyActionlogItemView myItemView = new MyActionlogItemView(injectionContext::get);
-		messages.getChildren().add(myItemView.getView());
-		messageView.add(myItemView);
+	void receiveDropMessages(Date siceDate) throws QblDropInvalidMessageSizeException, QblVersionMismatchException, QblSpoofedSenderException {
+		List<DropMessage> dropMessages = getDropMassages(siceDate);
 
-		injectionContext = new HashMap<>();
+		if (dropMessages == null) {
+			return;
```

Comment:
if getDropMessages doesn't ever return null (but either a list or an exception), you don't even need this code